types/multy/index.d.ts

```diff
@@ -4,17 +4,17 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
 
-import * as Koa from "koa";
-import * as busboy from "busboy";
+import * as Koa from 'koa';
+import * as busboy from 'busboy';
 
-declare module "koa" {
+declare module 'koa' {
     interface Request {
         body: any;
     }
 }
 
 declare namespace multy {
-    type Options = busboy.BusboyConfig;
+    type Options = busboy.Config;
```

Comment:
my comment in `v0` about aliases was intended for this use-case. 
Provide alias, if that is enough for backward compatible DT types.
if you remove v0, this package will require `package.json`, as it is 5 years old and
orks only with v0.2 of